app/javascript/app/components/ndcs/ndcs-enhancements-viz/ndcs-enhancements-viz-selectors.js

```diff
@@ -0,0 +1,282 @@
+      if (path.properties.layer !== PATH_LAYERS.ISLANDS) {
+        const { locations, legendBuckets } = indicator;
+
+        if (!locations) {
+          paths.push({
+            ...path,
+            countryStyles
+          });
+          return null;
+        }
+
+        const iso = path.properties && path.properties.id;
+        const isEuropeanCountry = europeanCountries.includes(iso);
+        const countryData = isEuropeanCountry
+          ? locations[europeSlug]
+          : locations[iso];
+
+        let style = countryStyles;
+        if (countryData && countryData.label_id) {
+          const legendIndex = legendBuckets[countryData.label_id].index;
+          const color = getColorByIndex(legendBuckets, legendIndex, MAP_COLORS);
+          style = {
+            ...countryStyles,
+            default: {
+              ...countryStyles.default,
+              fill: color,
+              fillOpacity: 1
+            },
+            hover: {
+              ...countryStyles.hover,
+              fill: color,
+              fillOpacity: 1
+            }
+          };
+        }
+
+        paths.push({
+          ...path,
+          style
+        });
+      }
+      return null;
+    });
+    return paths;
+  }
+);
+
+export const getLinkToDataExplorer = createSelector([getSearch], search => {
+  const section = 'ndc-content';
+  return generateLinkToDataExplorer(search, section);
+});
+
+//Chart data methods
+
+export const summarizeIndicators = createSelector(
+  [getIndicatorsParsed, getMapIndicator],
+  (indicators, indicator) => {
+    if (!indicator || !indicators) return null;
+    let summaryData = {};
+    //Retain functionality for showing submitted 2020 NDCs in case this becomes useful to display later
+    //ONLY planned 2020 NDCs currently displayed in component
+    ['planned', 'submitted'].forEach(type => {
+      summaryData[type] = {
+        countries: {
+          value: 0,
+          max: Object.keys(indicator.locations).length,
+          opts: {
+            color: getColorByIndex(
+              indicator.legendBuckets,
+              type == 'submitted' ? '1' : '2',
+              MAP_COLORS
+            ),
+            label: 'countries have stated their intention to submit a 2020 NDC'
+          }
+        },
+        emissions: {
+          value: 0,
+          max: 100,
+          opts: {
+            color: getColorByIndex(
+              indicator.legendBuckets,
+              type == 'submitted' ? '1' : '2',
+              MAP_COLORS
+            ),
+            suffix: '%',
+            label:
+              'of global emissions are represented by these countries (2015 emissions data)'
+          }
+        }
+      };
+    });
+    const emissionsIndicator = indicators.find(
+      indicator => indicator.value == 'ndce_ghg'
+    );
+    for (let l in indicator.locations) {
+      const location = indicator.locations[l];
+      var type =
+        location.label_id == 235
```

Comment:
As above I'm worried that hard-coding these ids might be causing the issue where I don't see any data.